### PR TITLE
[WIP] Known issues for 0.16.6

### PR DESCRIPTION
**Pull request by benjaoming**
Let's make it clear to everyone that we are aware of some things and working on them so they know what to expect!

**Ship early, ship often!** :)

### CODE REVIEW

**Comment by benjaoming** [comment]
This may not make it in time for the bundled docs, but that's not critical.. it'll go in the release announcement and in the online docs.

Any suggestions about other critical known issues I should add?

Mistakes?

File: docs/installguide/release_notes.rst [hunk 1]
```diff
@@ -19,6 +19,28 @@ Bug fixes
...
+
+Please note that issues with **content packs** are not related to the software
+itself but is being fixed and updated along side our release.
+
+Watch
+`dev@learningequality.org <https://groups.google.com/a/learningequality.org/forum/#!forum/dev>`__
+for announcements.
+
+ * **Content packs** 1800 questions (assessment items) inside exercises (English version) have been removed due to an upstream data bug. The count is lower in other languages. There are 29,839 assessment items, so it's 6% of questions (not exercises) that are missing. In terms of exercises, they are all there.
+ * **Content packs** ~1700 dubbed videos are missing in Non-English versions of the content packs, making several languages like Swahili, Xhosa, Zulu have empty content databases. Issue tracking in `content-pack-maker#28 <https://github.com/fle-internal/content-pack-maker/issues/28>`__. :url-issue:`5120` :url-issue:`5091`
```
Comment (aronasorman) [inline docs/installguide/release_notes.rst]: `Issue tracking in` -> `These issues can be tracked in`

File: docs/installguide/release_notes.rst [hunk 2]
```diff
@@ -19,6 +19,28 @@ Bug fixes
...
+^^^^^^^^^^^^
+
+Please note that issues with **content packs** are not related to the software
+itself but is being fixed and updated along side our release.
+
+Watch
+`dev@learningequality.org <https://groups.google.com/a/learningequality.org/forum/#!forum/dev>`__
+for announcements.
+
+ * **Content packs** 1800 questions (assessment items) inside exercises (English version) have been removed due to an upstream data bug. The count is lower in other languages. There are 29,839 assessment items, so it's 6% of questions (not exercises) that are missing. In terms of exercises, they are all there.
```
Comment (aronasorman) [inline docs/installguide/release_notes.rst]: `1800 questions (assessment items) inside exercises (English version) have been removed due to an upstream data bug`

= >

`1800 outdated questions (assessment items) inside exercises were removed. (...). All exercises are still there.

Comment (benjaoming) [inline docs/installguide/release_notes.rst]: Awesome, I also moved it from Known Issues to Bug Fixes!

**Comment by benjaoming** [comment]
Merging this after aron's review. Any further improvements are welcome but will not make it to embedded docs.

**Pull request merged**